frappe/core/doctype/package_import/package_import.py

```diff
@@ -55,11 +55,11 @@ def import_package(self):
 		for module in os.listdir(package_path):
 			module_path = os.path.join(package_path, module)
 			if os.path.isdir(module_path):
-				get_doc_files(files, module_path)
+				files = get_doc_files(files, module_path)
```

Comment:
isn't this fine as it is? we pass reference of `files` list to this function so it will just mutate the list in the function 🤔

![Screenshot 2022-06-06 at 11 05 48 PM](https://user-images.githubusercontent.com/32034600/172214711-1693b82f-5ade-49fb-943c-0cbae06ebbff.png)

Reply:
Yes, I guess that was the case originally, but after [this](https://github.com/frappe/frappe/commit/b2b391e90ae7fa82b4b0ff5a3eab8f2b9c6ba007) change it's the copy of `files` that's being worked upon in `get_doc_files`.

https://github.com/frappe/frappe/blob/df3601b67bc51e11024b6159b463f12acbeaaa52/frappe/model/sync.py#L112

Reply:
It's easier to read code when inputs aren't mutated anyway. :)